test/e2e/apimachinery/aggregator.go

```diff
@@ -202,7 +190,7 @@ func TestSampleAPIServer(f *framework.Framework, image, namespaceName string) {
 			"tls.key": context.apiserverKey,
 		},
 	}
-	_, err = client.CoreV1().Secrets(namespace).Create(secret)
+	_, err := client.CoreV1().Secrets(namespace).Create(secret)
```

Comment:
Fix for compile failure: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53479/pull-kubernetes-e2e-gce-bazel/33539/